hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -403,7 +408,7 @@ private boolean checkAllNodesAreEqual(NetworkTopology topology) {
   @VisibleForTesting
   protected DatanodeDetails chooseNodeFromNetworkTopology(
       NetworkTopology networkTopology, DatanodeDetails anchor,
-      List<DatanodeDetails> excludedNodes) {
+      List<DatanodeDetails> excludedNodes) throws SCMException {
```

Comment:
You can remove the `throws SCMException` in the method definition now, as it is no longer used.

Reply:
Updated.